app/assets/javascripts/app/notifications/notifications.module.js

```diff
@@ -0,0 +1,31 @@
+(function() {
+  'use strict';
+
+  angular
+    .module('council.notifications', [
+      'council.core',
+      'ngAnimate'
+    ])
+    .config(setupRoutes);
+
+  function setupRoutes($stateProvider) {
+    $stateProvider
+      .state('notifications', {
+        url: '/notifications',
+        abstract: true,
+        template: '<div ui-view class="u-fullSize"></div>'
+      })
+      .state('notifications.index', {
+        url: '',
+        views: {
+          '': {
+            controller: 'NotificationsCtrl as ctrl',
+            templateUrl: 'notifications/index.html'
+          },
+          'sidebar@notifications.index': {
```

Comment:
It's not. It was from a previous attempt of trying a sidebar, and I forgot to remove this.
I removed it now.